### PR TITLE
numa_config_with_auto_placement: remove dependency on cgget

### PR DESCRIPTION
**Pull request by dzhengfy**
Signed-off-by: Dan Zheng <dzheng@redhat.com>

### CODE REVIEW

**Comment by dzhengfy** [comment]
Before the fix:
TestFail: Unexpected failure during the test: Command 'cgget -g cpuset /machine.slice/machine-qemu\\x2d2\\x2davocado\\x2dvt
x2dvm1.scope/libvirt/emulator' failed.stdout: b''stderr: b'/bin/sh: line 1: cgget: command not found\n'

After the fix:
 (1/1) type_specific.io-github-autotest-libvirt.numa_config_with_auto_placement.memory_bind: PASS (47.75 s)
libvirt-8.3.0-1.el9.x86_64

File: libvirt/tests/src/numa/numa_config_with_auto_placement.py [hunk 1]
```diff
@@ -197,23 +197,18 @@ def check_cgget_output(test, cgget_message):
     if not slice_line:
         test.error("'machine-qemu' is not found in 'systemd-cgls cpuset' output")
     slice_line = slice_line.replace('\\', '\\\\')[2:]
+
+    cgtest = libvirt_cgroup.CgroupTest('')
+    is_cgroup2 = True if cgtest.is_cgroup_v2_enabled() else False
```
Comment (Yingshun) [inline libvirt/tests/src/numa/numa_config_with_auto_placement.py:202]: How about changing it as below:
```suggestion
    is_cgroup2 = cgtest.is_cgroup_v2_enabled()
```

Comment (chunfuwen) [inline libvirt/tests/src/numa/numa_config_with_auto_placement.py:202]: it looks like is_cgroup2 variable is local scope of this method and used only once, so maybe it is not necessary to declare it, just use  cgtest.is_cgroup_v2_enabled() directly

**Review by chunfuwen — COMMENTED** [review]
see inline comment

**Comment by dzhengfy** [comment]
(1/2) type_specific.io-github-autotest-libvirt.numa_config_with_auto_placement.default: PASS (57.48 s)
 (2/2) type_specific.io-github-autotest-libvirt.numa_config_with_auto_placement.memory_bind: PASS (48.25 s)
libvirt-8.4.0-1.el9.x86_64

**Review by chunfuwen — APPROVED** [review]
LGTM

**Review by Yingshun — APPROVED** [review]

**Pull request merged**